doc/articles/getting-started/counterapp/includes/include-mvux.md

```diff
@@ -13,35 +13,62 @@ As part of creating the application, we selected MVUX as the presentation framew
     }
     ```
 
-- Add the `Count` and `Step` properties, to the `MainModel` class. These fields both must be of type `IState<int>` and to initialize them we use `=> State.Value(...)`.
+- Add a new `partial record` above named `Countable`. This record will be responsible for updating our counter's properties all while ensuring immutability. Learn more about immutable records [here](https://platform.uno/docs/articles/external/uno.extensions/doc/Learn/Mvux/WorkingWithRecords.html#how-to-create-immutable-records).
 
     ```csharp
-    public IState<int> Count 
-        => State.Value(this, () => 0);
+    internal partial record Countable
+    {
+    }
+    ```
 
-    public IState<int> Step 
-        => State.Value(this, () => 1);
+- Add the `Count` and `Step` properties to the `Countable`'s primary constructor. 
+
+    ```csharp
+    internal partial record Countable(int Count, int Step)
+    {
+    }
     ```
 
-- Add a method called `IncrementCommand` to the `MainModel` that will increment the counter by the step size. The generated ViewModel of our `MainModel` will automatically re-expose as `ICommand` the `IncrementCommand` method.
+- Add an `Increment` method to the `Countable` record. The `with` operator allows us to [create a new instance](https://platform.uno/docs/articles/external/uno.extensions/doc/Learn/Mvux/WorkingWithRecords.html#updating-records) of the object.
 
     ```csharp
-    public ValueTask IncrementCommand(int Step)
-        => Count.Update(c => c + Step, CancellationToken.None);
+    public Countable Increment() => this with
+    {
+        Count = Count + Step
+    };
+    ```
+
+- Add the newly created `Countable` as a property in the `MainModel` class. The type must be `IState<Countable>` and we use `=> State.Value(...)` to initialize it.
+
+    ```csharp
+    public IState<Countable> Countable => State.Value(this, () => new Countable(0, 1));
+    ```
+
+- Add a method named `IncrementCommand` to the `MainModel` that will in turn call the `Countable`'s `Increment` method and therefore update the counter. The generated ViewModel of our `MainModel` will automatically re-expose as `ICommand` the `IncrementCommand` method.
```

Comment:
Is it important at this point to mention the Generated MainModel and the ICommand?

Reply:
good point, I think we could instead mention it here when we are databinding: https://github.com/unoplatform/uno/blob/7f009e1c3fa6e4acb07e92ece2cd4ef62524b0bf/doc/articles/getting-started/counterapp/get-started-counter-csharp-mvux.md?plain=1#L126

(and here) https://github.com/unoplatform/uno/blob/7f009e1c3fa6e4acb07e92ece2cd4ef62524b0bf/doc/articles/getting-started/counterapp/get-started-counter-xaml-mvux.md?plain=1#L122

Reply:
@ajpinedam I have removed the mention and added a reference to commands in MVUX instead